src/Tests.props

```diff
@@ -2,7 +2,7 @@
   <Import Project="$(MSBuildThisFileDirectory)Tests.local.props" Condition="Exists('$(MSBuildThisFileDirectory)Tests.local.props')" />
 
   <PropertyGroup>
-    <NoWarn>$(NoWarn);1591;IDE1006</NoWarn>
+    <NoWarn>$(NoWarn);1591;IDE1006;CS0618</NoWarn>
```

Comment:
This ignores Obsolete warnings in tests.  Ok?

Reply:
ok